hydra-redis/querying_mechanism.py

```diff
@@ -0,0 +1,184 @@
+import redis
+import hydra_graph as graph
+from hydrus.hydraspec import doc_maker
+import hydrus
+from collections_endpoint import CollectionEndpoints
+import urllib.request
+import json
+
+def final_file(url):
+    """ Open the given url and read and load the Json data."""
+    response = urllib.request.urlopen(url)
+    return json.loads(response.read().decode('utf-8'))
+
+def split_data(count,objects):
+    for obj in objects:
+        string = obj.decode('utf-8')
+        map_string = map(str.strip,string.split(','))
+        property_list = list(map_string)
+        check = property_list.pop()
+        property_list.append(check.replace("\x00",""))
```

Comment:
How it happens that this character `"\x00"` ends up here?

Reply:
`"\x00" is null character and In C it works also as a string terminator character. [Here](https://stackoverflow.com/questions/24948139/why-get-string-from-redis-will-miss-blank-space) is something related to this.